src/core/api/index.js

```diff
@@ -207,11 +211,37 @@ type FeaturedParams = {|
 |};
 
 export function featured({ api, filters, page }: FeaturedParams) {
+  const newFilters = { ...filters };
```

Comment:
This workaround code looks identical to that in `search()` so you should be calling it from a shared function. I would suggest giving it a name that stands out so that we are motivated to fix it soon :) Maybe like:

```js
const newFilters = fixFiltersForAndroidThemes({ filters, api });
```

Reply:
Fair enough :)